### ISSUE TITLE
An old bundler version is locked-in in procto.gemspec

### ISSUE THREAD

**Issue by ch1c0t** [snusnu/procto#1]
```
➜  procto git:(master) bundle
Updating https://github.com/rom-rb/devtools.git
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler (~> 1.6.1) ruby

  Current Bundler version:
    bundler (1.10.6)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?
Could not find gem 'bundler (~> 1.6.1) ruby in any of the sources
```

**Comment by backus** [comment]
This should be closed by #3 which bumps the development dependency

**Comment by mbj** [comment]
Closing because of #3 was merged.
